src/base/controller.hpp

```diff
@@ -106,6 +106,12 @@ namespace Base {
  * - "max_memory_mb" (int): Sets the maximum size of memory for a store.
  *      If a state needs more, an error is thrown. If set to 0, the maximum
  *      will be automatically set to the system memory size [Default: 0].
+ * - "parallel_experiments" (int): Explicitly set number of circuits that
+ *      are executed in parallel [Default: -1].
```

Comment:
The default value for automatic in the `max_parallel_*` is currently 0 instead of -1. Can you change them so that this is consistent for the different parallel variables (you'll need to update the doc string in the python backend files too).

Reply:
I changed it to 0.